playground/CosmosEndToEnd/CosmosEndToEnd.AppHost/Program.cs

```diff
@@ -12,7 +12,7 @@
 
 // This project is only added in playground projects to support development/debugging
 // of the dashboard. It is not required in end developer code. Comment out this code
-// to test end developer dashboard launch experience. Refer to WorkloadAttributes.cs
+// to test end developer dashboard launch experience. Refer to Directory.Build.props
 // for the path to the dashboard binary (defaults to a relative path to the artifacts
 // directory).
```

Comment:
Will `AspireDashboardDir` also be relative as the comment here suggests?

Reply:
Good point; I'll update the comment to reflect that the default is based on the DotNetOutputPath property.